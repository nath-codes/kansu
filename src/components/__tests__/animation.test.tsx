import 'jest-styled-components'
import React from 'react'

import Animation from '../animation'
import { renderWithTheme } from './helpers'

describe('Animation', () => {
  it('renders correctly', () => {
    const tree = renderWithTheme(
      <Animation width={100} height={100} />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
