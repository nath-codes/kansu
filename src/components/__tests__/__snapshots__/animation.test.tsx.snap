// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Animation renders correctly 1`] = `
<canvas
  className=""
/>
`;
